### PR TITLE
TF Refactor, Raffle Variables, and Documentation

### LINKED ISSUE #4: Delete User from TF CSV instead of store "a human"
https://github.com/Zielmann/TwitchBot-Public/blob/ca1cc607cedc84d2bb211e434dc304206f21c83d/Modules/tf.py#L87-L94

We can probably just delete their names instead of storing "a human" which takes up more space.

### PR DESCRIPTION
**Pull request by cai-michael**
- Raffle Variables are now set using a constructor
- TF is refactored to use JSON and dict
- Added Documentation to Methods

closes #4

### CODE REVIEW

File: Modules/tf.py [hunk 1]
```diff
@@ -3,11 +3,44 @@
...
 import settings
+import json
 from twitchio.ext.commands.core import cog
 from twitchio.ext.commands.core import command
 
-# Stores name/species pair into file
+
+def load_tf_list():
+    """
+    Loads the TF List into the variable tfList from "Data/tfList.json"
```
Comment (tsukikita) [inline Modules/tf.py:13]: Consider a bit more detail in the docstrings; i.e. I know Python is dynamically typed, but some idea of what a "tf list" is might be useful for developers. Looking at the code, I can see that it's a dictionary of species by name.

File: Modules/tf.py [hunk 2]
```diff
@@ -20,8 +53,23 @@ def writeSpeciesToCSV(name, species):
         for item in userDict:
             writer.writerow([item, userDict[item]])
 
-# Gets name/species pair from file. Return 'a human' if user not in list
+def getSpecies(name):
+    """
+    Returns the name's current species
```
Comment (tsukikita) [inline Modules/tf.py:58]: Similar to above, a bit more detail on the parameter would be helpful. In this case it's fairly clear that "name" should be a text value, but this isn't always the case.

File: Modules/rimworld.py [hunk 3]
```diff
@@ -40,6 +47,10 @@ def item_search(ctx):
     return response
 
 def event_search(ctx):
+    """
+    Searches the users Rimworld installation for events containing that keyword
```
Comment (tsukikita) [inline Modules/rimworld.py:51]: "Keyword" is not an argument -- Might be worth specifying where the keyword comes from in the docstring

File: Modules/rimworld.py [hunk 4]
```diff
@@ -57,6 +68,9 @@ def event_search(ctx):
     return response
 
 def event_detail_search(ctx):
+    """
+    Returns details about a specific event
```
Comment (tsukikita) [inline Modules/rimworld.py:72]: See above; event is not a parameter, where does this come from?

File: Modules/rimworld.py [hunk 5]
```diff
@@ -68,6 +82,9 @@ def event_detail_search(ctx):
     return response
 
 def item_detail_search(ctx):
+    """
+    Returns details about a specific item
```
Comment (tsukikita) [inline Modules/rimworld.py:86]: Same as above

**Comment by Zielmann** [comment]
FYI - The change to using a constructor for Raffle breaks the module import.

Exception has occurred: TypeError
__init__() takes 1 positional argument but 2 were given
  File "C:\Users\mattw\PythonProjects\TwitchBot-Public\bot.py", line 60, in <module>
    bot.load_module('Modules.raffle')

**Comment by cai-michael** [comment]
> FYI - The change to using a constructor for Raffle breaks the module import.
> 
> Exception has occurred: TypeError
> **init**() takes 1 positional argument but 2 were given
> File "C:\Users\mattw\PythonProjects\TwitchBot-Public\bot.py", line 60, in
> bot.load_module('Modules.raffle')

Reverted these changes

Added some more documentation past Tsuki's review

**Pull request merged**